### PR TITLE
AML Florence Accelerator & Bugfix

### PR DESCRIPTION
**Pull request by yvonnebarthp**
## Purpose

We created a Florence accelerator to approach model training and inference in a code-first manner
* It enables the user to bridge the gap between AML and AI Vision using a central yaml file, which gives the user more control over fine-tuning Florence models
* Allows for model training and inference 
* Key features: 
** Ability to run Florence finetuning against Azure ML
** Out of the box usage of AML datasets which are automatically registered in Azure AI Vision Studio
** Model training, tracking and metrics comparison in AML Dashboards
* Bugfix for 400 bad request error (InvalidRequest: AuthenticationKind is required)

## Does this introduce a breaking change?

```
[x] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: aml-pipeline/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+## Overview
```
Comment (pj-ms) [inline aml-pipeline/README.md:1]: Do you mind adding a pointer from the README.md at the root dir to this one with a brief description? Thanks

File: aml-pipeline/.amlignore [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## This file was auto generated by the Azure Machine Learning Studio. Please do not remove.
```
Comment (pj-ms) [inline aml-pipeline/.amlignore:1]: Please update the pkg version in setup.py

Comment (yvonnebarthp) [reply aml-pipeline/.amlignore:1]: Should we update to version '0.0.6'?

File: aml-pipeline/adjust_coco.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import argparse
```
[the remark this answers is not in the document]
Comment (yvonnebarthp) [reply aml-pipeline/adjust_coco.py:1]: Yes, we need this helper script to adjust custom image paths in the coco json (Described in the Readme).

**Review by pj-ms — APPROVED** [review]

**Pull request merged**